benchmark_runner/workloads/workloads_operations.py

```diff
@@ -392,6 +401,34 @@ def update_ci_status(self, status: str, ci_minutes_time: int, benchmark_operator
         metadata.update({'status': status, 'status#': status_dict[status], 'ci_minutes_time': ci_minutes_time, 'benchmark_operator_id': benchmark_operator_id, 'benchmark_wrapper_id': benchmark_wrapper_id, 'ocp_install_minutes_time': ocp_install_minutes_time, 'ocp_resource_install_minutes_time': ocp_resource_install_minutes_time})
         self._es_operations.upload_to_elasticsearch(index=es_index, data=metadata)
 
+    @logger_time_stamp
+    def set_bootstorm_vm_start_time(self, vm_name: str = ''):
+        """
+        This method capture boot storm when vm start
+        @return:
+        """
+        self.bootstorm_start_time[vm_name] = time.time()
+
+    @logger_time_stamp
+    def get_bootstorm_vm_end_time(self, vm_name: str):
+        """
+        This method return boot storm vm time in milliseconds
```

Comment:
Elapsed time, correct?  Then the comment should be "Return VM start time for specified VM, in milliseconds".

Reply:
Fixed: This method returns boot vm elapse time for specified VM in milliseconds